Sentiment.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+
+# fileName = input("Please enter the text file:")
+
+# Get words from the text file and remove words with less than 3 letters
+def get_sentiment(file_name):
```

Comment:
The instructions was clear. Your solution should accept a text excerpt not a file.